tests/foreman/cli/test_contentview.py

```diff
@@ -4809,74 +4653,64 @@ def test_positive_publish_promote_with_RH_ostree_and_other(self):
+            {
+                'name': rpm_repo_name,
+                'organization-id': module_manifest_org.id,
+                'product': PRDS['rhel'],
+            }
         )
-        cv = make_content_view({'organization-id': self.org['id']})
+        cv = make_content_view({'organization-id': module_manifest_org.id})
         # Associate repo to CV with names.
         ContentView.add_repository(
             {
                 'name': cv['name'],
-                'organization': self.org['name'],
+                'organization': module_manifest_org.name,
                 'product': PRDS['rhah'],
-                'repository': self.repo_name,
+                'repository': atomic_repo['name'],
             }
         )
         ContentView.add_repository(
             {
                 'name': cv['name'],
-                'organization': self.org['name'],
+                'organization': module_manifest_org.name,
                 'product': PRDS['rhel'],
                 'repository': rpm_repo_name,
             }
         )
         cv = ContentView.info({'id': cv['id']})
-        self.assertEqual(
-            cv['ostree-repositories'][0]['name'],
-            self.repo_name,
-            'RH Ostree Repo was not associated to CV',
-        )
-        self.assertEqual(
-            cv['yum-repositories'][0]['name'],
-            rpm_repo_name,
-            'RH rpm Repo was not associated to CV',
-        )
+        assert cv['ostree-repositories'][0]['name'] == atomic_repo['name']
+        assert cv['yum-repositories'][0]['name'] == rpm_repo_name
         ContentView.publish({'id': cv['id']})
         cv = ContentView.info({'id': cv['id']})
-        lc_env = make_lifecycle_environment({'organization-id': self.org['id']})
+        lc_env = make_lifecycle_environment({'organization-id': module_manifest_org.id})
         # Promote the Published version of CV to the next env
         ContentView.version_promote(
             {'id': cv['versions'][0]['id'], 'to-lifecycle-environment-id': lc_env['id']}
         )
         cv = ContentView.info({'id': cv['id']})
         environment = {'id': lc_env['id'], 'name': lc_env['name']}
-        self.assertIn(environment, cv['lifecycle-environments'])
+        assert environment in cv['lifecycle-environments']
 
 
-class ContentViewFileRepoTestCase(CLITestCase):
+class TestContentViewFileRepo:
     """Specific tests for Content Views with File Repositories containing
     arbitrary files
     """
 
-    @classmethod
-    def setUpClass(cls):
-        """Create a product and an org which can be re-used in tests."""
-        super().setUpClass()
-        cls.org = make_org()
-        cls.product = make_product({'organization-id': cls.org['id']})
-
-    def _make_file_repository_upload_contents(self, options=None):
+    def _make_file_repository_upload_contents(self, module_org, module_product, options=None):
```

Comment:
Given that these are pytest classes and not library classes, you don't necessarily need to name this as if it were a private method.

Reply:
True, but I'm just keeping what was originally there and making as little changes to what was there already.  I can still change this.

Reply:
Its optional, but was worth mentioning.